javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,13 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  // create an array to hold the new words with added suffix
+  var suffixed = [];
+  // loop over each each word given
+  for (var i = 0; i < words.length; i++) {
+    // add suffix to each word
+    // add the new word to the array
+    suffixed.push(words[i] += suffix);
```

Comment:
Oh no, this is bad news because you are changing the original array. You want to use the values of the array given as an argument, but you never want to change the original values.

Reply:
Got it, I'll correct it

Reply:
![Screen Shot 2022-05-20 at 10 38 30](https://user-images.githubusercontent.com/68994507/169582669-fd139276-042e-47b7-ab46-02c1895dc047.png)